app/main.py

```diff
@@ -14,14 +28,18 @@ def main():
     q = request.args.get('q')
     if not q:
         abort(400)
+    key = '_'.join([
+        conf.KEY_PREFIX,
+        hashlib.md5(str(q).encode('utf-8')).hexdigest()
```

Comment:
This actually isn't an error due to how I'm using unicode here—that's just for generating a cache key—it's that Bing's suggestion API doesn't appear to recognize/return results for it:

https://api.bing.com/osjson.aspx?query=%F0%9F%92%94

In that case, the traceback + empty response is the desired behavior.